extensions/sql-migration/src/dialog/skuRecommendationResults/SelectStorageAccountDialog.ts

```diff
@@ -365,6 +361,16 @@ export class SelectStorageAccountDialog {
 				CSSStyles: { ...DROPDOWN_CSS },
 			}).component();
 
+		this._disposables.push(
+			this._blobContainerDropdown.onValueChanged(async (value) => {
+				if (value && value !== 'undefined') {
```

Comment:
Don't put undefined in quotes, this way it is compering with the literal 'undefined' not value undefined. it should be

value !== undefined

Reply:
done